### ISSUE TITLE
Question regarding training setup

### ISSUE THREAD

**Issue by imankgoyal** [MarkFzp/act-plus-plus#34]
Hello,

Congrats on the nice work. I had a few questions:

- As far I understand, ACT for Aloha was trained for each task separately, but I am unsure about the same for the ACT model in mobile Aloha?

- If a single model is trained for all tasks in mobile Aloha, how does the model now which task to do? Is it just from the starting condition?

- Also, within the same task, are all the subtasks trained together. If so, I was a bit confused how would each subtask accuracy be calculated seperately as done in the paper. As I believe the model takes in continuous stream of input and it would be hard to stop in the middle and start. Maybe I am missing something.

Again congrats on the paper and looking forward to your response.

Best,
Ankit
